kong/db/migrations/core/024_370_to_380.lua

```diff
@@ -0,0 +1,22 @@
+return {
+  postgres = {
+    up = [[
+      DO $$
+      BEGIN
+      CREATE TABLE IF NOT EXISTS clustering_sync_version (
+        "version" SERIAL PRIMARY KEY
+      );
+      CREATE TABLE IF NOT EXISTS clustering_sync_delta (
+        "version" INT NOT NULL,
+        "type" TEXT NOT NULL,
+        "id" UUID NOT NULL,
```

Comment:
What will happen if there is an entity without an `id` attribute or with an `id` that isn't a UUID type? Is it possible to create such an entity in Kong via a 3rd party plugin dao, or will the metaschema prevent this condition?